cacheds3storage/__init__.py

```diff
@@ -11,28 +11,31 @@ def __init__(self, *args, **kwargs):
         self.local_storage = get_storage_class(
             'compressor.storage.CompressorFileStorage')()
 
-    # Fix for "ValueError: seek of closed file" problem with boto3
-    # https://github.com/jschneier/django-storages/issues/382#issuecomment-377174808
-    def _save_content(self, obj, content, parameters):
+    # https://github.com/jschneier/django-storages/issues/382
```

Comment:
If this is removed completely, will it just pick up the one in `django-storages`?

Reply:
Hmmmm, good point.. maybe the fix is already in I may be getting confused. Let me see if I can reproduce this locally to test that.

Reply:
Okay, so when testing this locally (actually with `./manage.py collectstatic`):

When I remove the _save/_save_content override completely, I get the same error:
```
  File "/home/nik/src/d/mediathread/ve/lib64/python3.7/site-packages/django/core/files/base.py", line 55, in chunks
    self.seek(0)
ValueError: seek of closed file
```

With this change.... well it's still running and not showing any output yet. I'll see what happens, but looks promising.

Reply:
Okay, yes. It's actually copying the files successfully with this override... which unfortunately still seems necessary.